### PR TITLE
Integrate concept graphs api

### PR DESCRIPTION
**Pull request by fmatthies**

### CODE REVIEW

File: schemas/top-api.yml [hunk 1]
```diff
@@ -438,6 +438,155 @@ paths:
...
+      tags:
+        - conceptgraphs
+  /conceptgraphs/statistics:
+    parameters:
+      - $ref: '#/components/parameters/include'
+    get:
+      operationId: getConceptGraphStatistics
+      summary: Gets an overview of the concept graphs for a process
+      parameters:
+        - name: process
```
Comment (ChristophB) [inline schemas/top-api.yml:522]: What happens if this parameter is missing? Is it required and should be converted to a path parameter?

Comment (ChristophB) [reply schemas/top-api.yml:522]: Also: there might be no process for a given ID. This should return 404.

File: schemas/top-api.yml [hunk 2]
```diff
@@ -438,6 +438,155 @@ paths:
...
+                clusteringConfig:
+                  type: string
+                  format: binary
+                graphConfig:
+                  type: string
+                  format: binary
+              required:
+                - data
+      responses:
+        '200':
```
Comment (ChristophB) [inline schemas/top-api.yml:495]: Why there are both 200 and 202 responses?

File: schemas/top-api.yml (hunk 1 above)
Comment (fmatthies) [reply schemas/top-api.yml:522]: if this parameter is missing, it tries to fetch a process named  'default' (that's also true for creation etc.). If no process is available it returns json with an error message. But there should be a 404 response as well...

Comment (ChristophB) [reply schemas/top-api.yml:522]: I've added 404 for this specific case in [c310534](https://github.com/Onto-Med/top-api/pull/163/commits/c310534410bb952b7363f5bb7c96c8f81adbd5d7)

File: schemas/top-api.yml (hunk 2 above)
Comment (fmatthies) [reply schemas/top-api.yml:495]: You can start the pipeline process in such a way that it either waits for it being finished (you get a 200 response with graph statistics) or it just starts in the background and returns a 202 (with pipeline status response)

Comment (ChristophB) [reply schemas/top-api.yml:495]: Is this controlled with query parameter "returnStatistics"?

File: schemas/top-api.yml (hunk 1 above)
Comment (ChristophB) [reply schemas/top-api.yml:523]: Parameter `process` now has a default value 'default'.

File: schemas/top-api.yml (hunk 2 above)
Comment (ChristophB) [reply schemas/top-api.yml:496]: Documentation has been updated accordingly.

**Review by ChristophB — APPROVED** [review]

**Pull request merged**